ops/_vendor/websocket/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
The code under `ops/_vendor/websocket` doesn't need to be reviewed -- as mentioned above, it's external code copied directly from the [websocket-client](https://github.com/websocket-client/websocket-client) library at `v1.2.1`.